packages/mobile/routes/onboarding/views/CongratulationsView.svelte

```diff
@@ -1,12 +1,14 @@
 <script lang="typescript">
     import { OnboardingLayout } from '../../../components'
     import { Button, Icon, Text } from 'shared/components'
+    import { hasCompletedAppSetup } from '@core/app'
     import { onboardingRouter } from '@core/router'
     import { localize } from '@core/i18n'
     import { completeOnboardingProcess } from '@contexts/onboarding'
 
     function onContinueClick(): Promise<void> {
         completeOnboardingProcess()
+        hasCompletedAppSetup.set(true)
```

Comment:
`hasCompletedAppSetup` we shouldnt be setting this here imo, the variable is a bit confusing because is not about finalizing a full profile, but rather something like skip legal and appearance & language screens, so because we skip some of these screens in mobile, we should set this to true in the last view inside `app-setup`, which atm is simply the welcome page. Lets add it there for now in the continue click.
To understand the usage of this store, please refer to `packages/shared/lib/core/router/onboarding-router.ts#143`